src/core_atmosphere/physics/physics_wrf/module_cu_tiedtke.F

```diff
@@ -254,12 +254,9 @@ subroutine cu_tiedtke(                                    &
 ! to determine at run-time whether a particular tracer is in
 ! use or not.
 !
-     logical, optional ::                                    &
-                                                   f_qv      &
-                                                  ,f_qc      &
-                                                  ,f_qr      &
-                                                  ,f_qi      &
-                                                  ,f_qs
+     logical, optional ::                               &
+                                        f_qc                            &
```

Comment:
Laura would spank us for for this misalignment of the ampersands, but maybe we didn't do?